SQL/advanced.sql

```diff
@@ -376,3 +376,32 @@ CREATE VIEW booking_costs_today AS (
   ORDER BY
     "cost" DESC
 );
+
+
+----------------------------------------------------------------------------------------
+/*
+STORED FUNCTIONS
+*/
+
+-- passing parameters
+CREATE OR REPLACE PROCEDURE remove_member(
+  member_id integer
+)
+AS $$
+BEGIN
+  DELETE FROM members WHERE memid = member_id;
+  IF NOT FOUND THEN
+      RAISE NOTICE 'No member with specified id found';
+      RETURN;
```

Comment:
just for clarity, why do you need RETURN here?

Reply:
Indeed, no need :)
Will be removed in the next commit.